test/extensions/filters/http/ext_proc/ordering_test.cc

```diff
@@ -87,13 +87,13 @@ class OrderingTest : public testing::Test {
   // Send data through the filter as if we are the proxy
 
   void sendRequestHeadersGet(bool expect_callback) {
-    HttpTestUtility::addDefaultHeaders(request_headers_, "GET");
+    HttpTestUtility::addDefaultHeaders(request_headers_);
     EXPECT_EQ(expect_callback ? FilterHeadersStatus::StopIteration : FilterHeadersStatus::Continue,
               filter_->decodeHeaders(request_headers_, true));
   }
 
   void sendRequestHeadersPost(bool expect_callback) {
-    HttpTestUtility::addDefaultHeaders(request_headers_, "POST");
+    HttpTestUtility::addDefaultHeaders(request_headers_);
```

Comment:
Seems like these are supposed to differentiate between GET and POST, do we need to set the method header after the fact here?

Reply:
Fixed.